libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/RestRequest.java

```diff
@@ -320,6 +321,24 @@ public JSONObject getRequestBodyAsJson() {
 		return additionalHttpHeaders;
 	}
 
+    /**
+     * Returns whether the SDK should attempt to refresh tokens if the service returns HTTP 403.
+     *
+     * @return True - if the SDK should refresh on HTTP 403, False - otherwise.
+     */
+	public boolean getShouldRefreshOn403() {
+	    return shouldRefreshOn403;
+    }
+
+    /**
+     * Sets whether the SDK should attempt to refresh tokens if the service returns HTTP 403.
+     *
+     * @param shouldRefreshOn403 True - if the SDK should refresh on HTTP 403, False - otherwise.
+     */
+	public synchronized void setShouldRefreshOn403(boolean shouldRefreshOn403) {
```

Comment:
The caller should call this method to override default behavior on a `RestRequest` object.